tests/testlib/users.py

```diff
@@ -27,7 +27,7 @@ def _mk_user_obj(  # type:ignore[no-untyped-def]
     if password not in precomputed_hashes:
         raise ValueError("Add your hash to precomputed_hashes")
 
-    user = {
+    user: dict = {
```

Comment:
Is this needed?

Reply:
I find predicting `mypy` and friends to be quite difficult.  If these tools do not require it then it can be dropped.

Reply:
Removed.

Reply:
`mypy` fails when it is removed.